geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/GeodeRedisServer.java

```diff
@@ -94,9 +97,9 @@ public GeodeRedisServer(String bindAddress, int port, InternalCache cache) {
     nettyRedisServer = new NettyRedisServer(() -> cache.getInternalDistributedSystem().getConfig(),
         regionProvider, pubSub,
         this::allowUnsupportedCommands, this::shutdown, port, bindAddress, redisStats,
-        redisCommandExecutor, member);
+        redisCommandExecutor, member, commandHelper);
 
-    BucketInfoRetrievalFunction.register(bindAddress, nettyRedisServer.getPort());
+    RedisMemberInfoRetrievalFunction.register(bindAddress, nettyRedisServer.getPort());
```

Comment:
I think there is a race condition where the region might be created but the function is not registered yet ... that might lead to an error executing the function. Maybe we need register the function earlier and set the bind address & port later, or maybe handle that sort of error when we execute the function?

Reply:
This should be handled now.